tests/dev_env_tests/test_get_node_info.py

```diff
@@ -58,96 +58,3 @@ def test_get_node_info(node_id, expected_node_info):
     # datasets_per_data_model is tested elsewhere.
     assert node_info.id == expected_node_info.id
     assert node_info.role == expected_node_info.role
-
```

Comment:
Why are these tests gone?

Reply:
I looked into changing the tests, but as you said, in MIP-263 they were reworked, which is active in parallel, so there is no point for me to work on those as well...